cmd/cli/sign.go

```diff
@@ -61,7 +62,8 @@ func (a *annotationsMap) String() string {
 func Sign() *ffcli.Command {
 	var (
 		flagset     = flag.NewFlagSet("cosign sign", flag.ExitOnError)
-		key         = flagset.String("key", "", "path to the private key")
+		privKey     = flagset.String("key", "", "path to the private key")
+		publicKey   = flagset.String("public-key", "", "path to the public key")
```

Comment:
ooh, one thing we might be able to do is extract the public key from the private one to save the flag. The crypto private key interface has a .Public method on it.

Reply:
cool that's way better